main.go

```diff
@@ -134,23 +131,26 @@ func main() {
 	// assign global Verbose variable after arguments have been parsed
 	linepipes.Verbose = *fVerbose
 
-	var err error
+	reqtraqConfig, err := config.ParseConfig(repos.BaseRepoPath())
+	if err != nil {
+		log.Fatal("Error parsing `reqtraq_config.json` file in current repo:", err)
+	}
 
 	switch command {
 	case "nextid":
-		err = nextId(filename)
+		err = nextId(filename, &reqtraqConfig)
```

Comment:
Yes, I did not want to copy all Documents and configurations all over the place, given it could be quire large overall (containing all repos, documents, vectors with code files and test files, etc...). Would you suggest to change it?

Reply:
no, just personal preference (readability vs performance), I'm not convinced it makes a great difference performance wise but I have no evidence and no strong objection =)